compiler/luci/service/src/CircleShapeSignatureInferenceHelper.cpp

```diff
@@ -46,14 +46,24 @@ namespace ssinf
 
 ShapeSignature reduced_signature(const loco::Node *node, const loco::Node *indices, bool keep_dims)
 {
+  LOGGER(l);
+
+  ShapeSignature input_signature;
   ShapeSignature output_signature;
 
   auto circle_node = loco::must_cast<const luci::CircleNode *>(node);
-  auto circle_indices = loco::must_cast<const luci::CircleNode *>(indices);
-  auto input_signature = circle_node->shape_signature();
+
```

Comment:
maybe remove this line to follow below `auto reduction_indices = ` line?

Reply:
or go as-is...

Reply:
You mean following modification?
```suggestion
```
If so, there's no reason to refuse :-)